aries-site/src/examples/templates/filtering/FilteringWithSelect.js

```diff
@@ -33,322 +28,41 @@ const allData = [
-          }}
-          // target is for demo purposes only, remove in production
-          target={target}
+      <Grid height={{ min: 'medium' }}>
+        <Data
+          data={allData}
+          views={[
+            { name: 'All', properties: { status: ['Ready', 'Paused'] } },
+            { name: 'Paused', properties: { status: ['Paused'] } },
+            { name: 'Ready', properties: { status: ['Ready'] } },
+          ]}
         >
-          <Box
-            width={{ min: 'medium' }}
-            pad={{ horizontal: 'medium', vertical: 'medium' }}
-            gap="medium"
-            fill="vertical"
-          >
-            <Header>
-              <Heading as="p" color="text-strong" margin="none">
-                Filters
-              </Heading>
-              <Button
-                icon={<FormClose />}
-                onClick={() => {
-                  filterData(allData, previousFilters);
-                  restoreValues(previousValues);
-                  setShowLayer(!showLayer);
-                }}
-              />
-            </Header>
-            <Box pad="xsmall" overflow="auto" flex>
-              {content}
-            </Box>
-            <Box align="center" direction="row" gap="small">
-              <Button
-                label="Apply filters"
-                onClick={() => {
-                  const nextData = filterData(allData, filters);
-                  setData(nextData);
-                  setShowLayer(!showLayer);
-                }}
-                primary
-              />
-              <Button
-                label="Reset filters"
-                onClick={() => {
-                  resetFilters();
-                  setShowLayer(!showLayer);
-                }}
-                secondary
-              />
-            </Box>
-          </Box>
-        </Layer>
-      )}
-    </>
-  );
-};
-
-Filters.propTypes = {
-  filters: PropTypes.shape({}).isRequired,
-  setFilters: PropTypes.func.isRequired,
-  filtering: PropTypes.bool.isRequired,
-  filterData: PropTypes.func.isRequired,
-  setData: PropTypes.func.isRequired,
-  setFiltering: PropTypes.func.isRequired,
-  setSearch: PropTypes.func.isRequired,
-  // target is for demo purposes only, remove in production
-  target: PropTypes.object,
-};
-
-const RecordSummary = ({ data, filtering }) => (
-  <Box direction="row" gap="xxsmall">
-    <Text size="small" weight="bold">
-      {data.length}
-    </Text>
-    <Text size="small">
-      {filtering ? `result${data.length > 1 ? 's' : ''} of ` : 'items'}
-    </Text>
-    {filtering && (
-      <Box direction="row" gap="xxsmall">
-        <Text size="small" weight="bold">
-          {allData.length}
-        </Text>
-        <Text size="small">{`item${allData.length > 1 ? 's' : ''}`}</Text>
-      </Box>
-    )}
-  </Box>
+          <Toolbar>
+            <DataSearch responsive />
+            <DataView />
+          </Toolbar>
+          <DataSummary />
+          <Results />
```

Comment:
Can you file an issue for the following?

Explore `aria-describedby` attribute to reference the Heading. For example: https://github.com/grommet/hpe-design-system/blob/60cec71c8df5844dc9ed855b2aef32e6d75f6200/aries-site/src/examples/components/datatable/DataTableResizeColumnsExample.js#L198

Reply:
Thank you.